202009.0/41a02e12-95cd-4359-a632-42ce2e2a8a81.md

```diff
@@ -12,20 +12,32 @@ This document describes how to configure debugging of Spryker in Docker.
 [Xdebug](https://xdebug.org) is the default debugging tool for Spryker in Docker. To enable Xdebug, run the command:
 ```bash
 docker/sdk {run|start|up} -x
-``` 
+```
 
 ## Configuring Xdebug in PhpStorm
 
 This section describes the required configuration for Xdebug in PhpStorm.
 
+### Prerequisites
+
+1. PHPStorm - Early Access Program(EAP) preferred
+ - You can download PHPStorm through the [Jetbrains Toolbox](https://www.jetbrains.com/toolbox-app/)
+2. Xdebug 
+ - You can install and verify it [here](https://xdebug.org/docs/install)
+3. [Xdebug extension](https://www.jetbrains.com/help/phpstorm/2021.1/browser-debugging-extensions.html)
+ - set the debug key to `PHPSTORM`
+
 ### Configuring Xdebug
 
 To configure Xdebug in PhpStorm:
-1. Go to **Preferences** > **Languages & Frameworks** > **PHP** > **Debug**.
+
+1. Open your preferences, either from the main dropdown or with `Cmd/Ctrl + ,`
+      1. Inside the preferences pane go to **PHP** > **Debug**.
 
 2. In the *Xdebug* section:
 
       1. Depending on your requirements, enter a **Debug port**.
+            - Xdebug 2 uses port 9000, Xdebug 3 uses port 9003 you can enter both `9000,9003`
```

Comment:
yep, spent some time understanding this using only port 9000...
thanks for this update!